doc: Add known issue NCSDK-25144 (#13723)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to :ref:`nrfxlib:crypto`.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0

> NCSDK-25144: Selecting :kconfig:option:`CONFIG_SECURE_BOOT_CRYPTO` link will link nrf_cc310_bl into the main application.

Reviewer: ```suggestion
NCSDK-25144: Enabling :kconfig:option:`CONFIG_SECURE_BOOT_CRYPTO` will link nrf_cc310_bl into the main application.
```

> NCSDK-25144: Selecting :kconfig:option:`CONFIG_SECURE_BOOT_CRYPTO` link will link nrf_cc310_bl into the main application.
> 

Reviewer 2: ```suggestion
NCSDK-25144: Enabling Kconfig option :kconfig:option:`CONFIG_SECURE_BOOT_CRYPTO` links nrf_cc310_bl into the main application
```

>   Configuring usage of secure boot with :kconfig:option:`CONFIG_SECURE_BOOT_CRYPTO` will link the bootloader library nrf_cc310_bl into the main application,
>   which is invalid. This will make crypto operations such as ECDSA fail.

Reviewer 2: ```suggestion
  Configuring secure boot with Kconfig option :kconfig:option:`CONFIG_SECURE_BOOT_CRYPTO` links the bootloader library ``nrf_cc310_bl`` into the main application, which is invalid.
  This makes crypto operations, such as ECDSA, fail.
```

>   **Workaround:** Make sure :kconfig:option:`CONFIG_SECURE_BOOT_CRYPTO` is only set when building the b0 image.

Reviewer 2: ```suggestion
  **Workaround:** Make sure the Kconfig option :kconfig:option:`CONFIG_SECURE_BOOT_CRYPTO` is only set when building the ``b0`` image.
```
